### PR TITLE
Debriefing fields

### PR DESCRIPTION
**Pull request by gilleswittenberg**

### CODE REVIEW

File: src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx [hunk 1]
```diff
@@ -119,18 +125,8 @@ const DefinitionList: React.FC<DLProps> = ({ thread, showDate }) => {
...
-            { Object.keys(thread.event_values ?? {}).map((key, index) => (
-              <div key={index}>
-                <dt>{key}</dt>
-                <dd>{ value?.[key] }</dd>
-              </div>
-            ))
-            }
-          </>
-          :
+        { showDate && thread.date_created && <div><dt>Datum</dt><dd>{ displayDate(thread.date_created) }</dd></div> }
```
Comment (remcohoff) [inline src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx:128]: `thread.date_created` has to be `value.start_time`

File: src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx [hunk 2]
```diff
@@ -188,6 +184,31 @@ const DefinitionList: React.FC<DLProps> = ({ thread, showDate }) => {
               </div>
             }
           </>
+          : thread.type === "DEBRIEFING" ?
+          <>
+            <div>
+              <dt>Auteur</dt>
+              <dd>{ value.author }</dd>
+            </div>
+            <div>
```
Comment (remcohoff) [inline src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx:193]: No checks?

**Review by remcohoff — APPROVED** [review]

File: src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx (hunk 2 above)
Comment (gilleswittenberg) [reply src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx:193]: For `undefined`?

Comment (remcohoff) [reply src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx:193]: Yes

Comment (gilleswittenberg) [reply src/app/features/cases/components/organisms/CaseTimeline/TimelineCaseEvent.tsx:193]: It's always a string according to the generated types

**Pull request merged**